src/plugins/utils/Utils.ts

```diff
@@ -248,26 +248,25 @@ class Utils {
 
     static tryRequire(name: string, paths?: string[]): any {
         try {
-            let resolvedPath;
-            if (paths !== undefined) {
-                resolvedPath = customReq.resolve(name, { paths });
+            if (paths !== undefined && paths.indexOf(process.env.LAMBDA_TASK_ROOT) === -1) {
+                paths.push(process.env.LAMBDA_TASK_ROOT);
             } else {
-                resolvedPath = customReq.resolve(name);
+                paths = [process.env.LAMBDA_TASK_ROOT];
```

Comment:
If `/var/task` is already in the paths with others, this else statement overrides all of them with only `/var/task`